### ISSUE TITLE
Deprecation error in urlutils

### ISSUE THREAD

**Issue by dmitrizagidulin** [anvilresearch/oidc-rp#17]
At the moment, `urlutils@0.0.3` is giving a deprecation warning on installation:

```
npm WARN deprecated URIjs@1.16.1: package renamed to "urijs" (lower-case), please update accordingly

...
│ └─┬ oidc-rp@0.1.0
│   ├─┬ urlutils@0.0.3
│   │ └── URIjs@1.16.1
```
We should either make a PR upstream, to https://github.com/cofounders/urlutils (to rename the package), or if that lib is not maintained, ponder another one.

**Comment by dmitrizagidulin** [comment]
@christiansmith - I propose we switch from `urlutils` to [`whatwg-url`](https://www.npmjs.com/package/whatwg-url) (much more actively maintained and up to spec. I'll make a PR if you agree.

**Comment by dmitrizagidulin** [comment]
Er, wait, n/m.. the export signature is different on those two libs. (This is why we can't have nice things.)

```js
const URL = require('urlutils')

// vs

const { URL } = require('whatwg-url')
```

Argh. I'm not sure how to make the latter work isomorphically with Webpack

**Comment by dmitrizagidulin** [comment]
Posted a StackOverflow question about this, meanwhile: https://stackoverflow.com/questions/43101644/excluding-webpack-externals-with-library-components-fragments

**Comment by christiansmith** [comment]
Replacing seems like a good plan. Will accept PR with passing tests.

**Comment by dmitrizagidulin** [comment]
Fixed in PR #39, closing.
